cdk/cdk_stack.py

```diff
@@ -30,8 +39,8 @@ def __init__(self, scope: cdk.App, id: str, **kwargs) -> None:
             code=_lambda.Code.from_asset('lambda'),
             environment={
                 'BUCKET_NAME': apidata_bucket.bucket_name,
-                'API_ENDPOINT': 'http://api.openweathermap.org/data/2.5/air_pollution/history?lat=51.5098&lon=-0.1180&start'
-                                '=1606266000&end=dynamic&appid=57e5f883d398a3a11dd65e86c5909df4'
+                'API_ENDPOINT': f'http://api.openweathermap.org/data/2.5/air_pollution/history?lat={{lat}}&lon={{lon}}&start'
+                                f'=1672531200&end=dynamic&appid={secret_value}'
```

Comment:
Are you sure secret_value is a string?

Reply:
Yep, it takes in the string value stored to the secret key I manually created in Secrets Manager console.